src/main/java/com/launchdarkly/client/FeatureFlag.java

```diff
@@ -175,12 +174,10 @@ VariationOrRollout getFallthrough() {
   static class EvalResult {
     private JsonElement value;
     private List<FeatureRequestEvent> prerequisiteEvents;
-    private Set<String> visitedFeatureKeys;
 
-    private EvalResult(JsonElement value, List<FeatureRequestEvent> prerequisiteEvents, Set<String> visitedFeatureKeys) {
+    private EvalResult(JsonElement value, List<FeatureRequestEvent> prerequisiteEvents) {
```

Comment:
For the internal method yes, but we still need to return 2 objects (value + prereq events)
I removed the use of the EvalResult class from the internal method- hopefully that makes things easier.